python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -3249,3 +3251,23 @@ def forward(self, input, boundaries):
 @register_test_case(module_factory=lambda: BucketizeTensorStaticFloatModule())
 def BucketizeTensorStaticFloatModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(15, 17), torch.sort(tu.rand(16)).values)
+
+
+# ==============================================================================
+
+
+class GLUModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True)
+    ])
+    def forward(self, x):
+        return torch.ops.aten.glu(x, dim=-1)
```

Comment:
as discussed, can you please add a test that exercises this lowering through the public [torch api](https://pytorch.org/docs/stable/generated/torch.nn.GLU.html)